code/modules/cargo/bounties/botany.dm

```diff
@@ -177,18 +169,62 @@
 	bonus_desc = "Wear protection when handling them."
 	foodtype = "cheese"
 
-/datum/bounty/item/botany/pineapples
-	name = "Pineapples"
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/pineapple)
-	bonus_desc = "Not for human consumption."
-	foodtype = "ashtray"
+//Harder Stuff (3600-7200)
 
-/datum/bounty/item/botany/tomato
-	name = "Tomatoes"
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/tomato)
-	exclude_types = list(/obj/item/reagent_containers/food/snacks/grown/tomato/blue)
+// /datum/bounty/item/botany/kudzu
+// 	name = "Kudzu Pods"
+// 	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/kudzupod)
+// 	bonus_desc = "Store in a dry, dark place."
+// 	multiplier = 4
+
+/datum/bounty/item/botany/bungopit
+	name = "Bungo Pits"
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/bungopit)
+	bonus_desc = "Heartbreaker."
+	multiplier = 4
+
+/datum/bounty/item/botany/ambrosia_gaia
+	name = "Ambrosia Gaia Leaves"
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/ambrosia/gaia)
+	multiplier = 4
+	foodtype = "stew"
+
+/datum/bounty/item/botany/apple_golden
+	name = "Golden Apples"
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/apple/gold)
+	multiplier = 4
+	foodtype = "dessert"
 
 /datum/bounty/item/botany/tomato_bluespace
 	name = "Bluespace Tomatoes"
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/tomato/blue/bluespace)
 	multiplier = 4
+
+/datum/bounty/item/botany/cannabis
+	name = "Cannabis Leaves"
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/grown/cannabis)
+	exclude_types = list(/obj/item/reagent_containers/food/snacks/grown/cannabis/white, /obj/item/reagent_containers/food/snacks/grown/cannabis/death, /obj/item/reagent_containers/food/snacks/grown/cannabis/ultimate)
+	multiplier = 4 //hush money
```

Comment:
Could the value be lowered by 1? All it needs to be accessed is a hacked seed vendor and it's a little weird it has the same multiplier of gaia & co.
Same for life and death weed, omega is fine.

Reply:
This was the value beforehand, l don't agree with changing the reward for Life/Death/Omega weed as it's RNG as to which one you get. You -could- get the one you need, or you -could- mutate the other two several times.

Reply:
well, the omega weed bounty could get a little buff, ngl.